### PR TITLE
added params benchmarks

### PR DESCRIPTION
**Pull request by asmyasnikov**
```
BenchmarkParams/ParamsBuilder-12         	 1148254	       985.3 ns/op	     846 B/op	      18 allocs/op
BenchmarkParams/RawProtobuf-12           	 2557124	       472.5 ns/op	     625 B/op	      14 allocs/op
BenchmarkParams/ParamsFromMap-12         	  755361	      1434 ns/op	    1241 B/op	      34 allocs/op
BenchmarkParams/table/types-12           	 1000000	      1137 ns/op	    1031 B/op	      18 allocs/op
```

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [x] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

### CODE REVIEW

File: internal/bind/params.go [hunk 1]
```diff
@@ -23,8 +24,31 @@ var (
 	errMultipleQueryParameters = errors.New("only one query arg *table.QueryParameters allowed")
 )
 
+func asUUID(v interface{}) (*uuid.UUID, bool) {
+	if _, ok := v.(interface {
```
Comment (rekby) [inline internal/bind/params.go:28]: Do you need check the interface really? Few lines below you check concrete types

File: topic/topictypes/topictypes_test.go [hunk 2]
```diff
@@ -152,7 +152,6 @@ func TestTopicDescriptionFromRaw(t *testing.T) {
 	}
 
 	for _, v := range testData {
```
Comment (asmyasnikov) [inline topic/topictypes/topictypes_test.go:154]:
```suggestion
	v := v
	for _, v := range testData {
```

File: topic/topictypes/topictypes_test.go [hunk 3]
```diff
@@ -322,7 +321,6 @@ func TestTopicConsumerDescriptionFromRaw(t *testing.T) {
 		},
 	}
 	for _, v := range testData {
```
Comment (asmyasnikov) [inline topic/topictypes/topictypes_test.go:323]:
```suggestion
	for _, v := range testData {
	v := v
```

File: topic/topictypes/topictypes_test.go [hunk 4]
```diff
@@ -152,7 +152,7 @@ func TestTopicDescriptionFromRaw(t *testing.T) {
 	}
 
 	for _, v := range testData {
-		v := v
+	v := v
```
Comment (asmyasnikov) [inline topic/topictypes/topictypes_test.go:155]:
```suggestion
		v := v
```

File: topic/topictypes/topictypes_test.go [hunk 5]
```diff
@@ -322,7 +322,7 @@ func TestTopicConsumerDescriptionFromRaw(t *testing.T) {
 		},
 	}
 	for _, v := range testData {
-		v := v
+	v := v
```
Comment (asmyasnikov) [inline topic/topictypes/topictypes_test.go:325]:
```suggestion
		v := v
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,6 @@
+* Refactored golang types mapping into ydb types using `ydb.ParamsFromMap` and `database/sql` query arguments
+* Small braking change: type mapping for `ydb.ParamsFromMap` and `database/sql` type `uuid.UUID` changed from ydb type `Text` to ydb type `UUID`
```
Comment (asmyasnikov) [inline CHANGELOG.md:2]:
```suggestion
* Small breaking change: type mapping for `ydb.ParamsFromMap` and `database/sql` type `uuid.UUID` changed from ydb type `Text` to ydb type `UUID`
```

File: internal/bind/params.go [hunk 7]
```diff
@@ -23,128 +24,316 @@ var (
...
 
+// Benchmarking of asUUID
+//
+// explicit UUID   	 2566760	       462.1 ns/op	      32 B/op	       2 allocs/op
+// asUUID        	 2103366	       595.9 ns/op	      48 B/op	       3 allocs/op
+func asUUID(v interface{}) (value.Value, bool) {
+	if _, ok := v.(interface {
+		URN() string
+		ClockSequence() int
+		ID() uint32
```
Comment (asmyasnikov) [inline internal/bind/params.go:35]:
```suggestion
		URN() string
```

File: internal/bind/params.go [hunk 8]
```diff
@@ -23,128 +24,316 @@ var (
 	errMultipleQueryParameters = errors.New("only one query arg *table.QueryParameters allowed")
 )
 
+// Benchmarking of asUUID
+//
+// explicit UUID   	 2566760	       462.1 ns/op	      32 B/op	       2 allocs/op
+// asUUID        	 2103366	       595.9 ns/op	      48 B/op	       3 allocs/op
+func asUUID(v interface{}) (value.Value, bool) {
+	if _, ok := v.(interface {
```
Comment (asmyasnikov) [inline internal/bind/params.go:32]:
```suggestion
	// explicit casting of type [16]byte to uuid.UUID will success, 
	// but casting of [16]byte to some interface with  methods from uuid.UUID will failed
	if _, ok := v.(interface {
```

**Pull request merged**